src/main/java/com/rocketden/main/dto/game/PlayerDto.java

```diff
@@ -10,8 +10,9 @@
 @Getter
 @Setter
 public class PlayerDto {
-    private UserDto userDto;
-    private PlayerCodeDto playerCode;
+    private UserDto user;
+    private String code;
+    private String language;
```

Comment:
Mainly I was thinking that a flattened structure for the frontend would be easier/cleaner to use (so you can call `player.code` instead of say `player.playerCode.code`) - however I'm open to thoughts on this design discussion

Reply:
The reason I created the object `PlayerCode` to include `code` and `language` is because I thought there may be other variables we include in the future, mainly because I wasn't really familiar with how this code would be transmitted back and forth. However, I'm not sure if this concern still holds up -- we can talk about this during the standup.

If we do end up not using `PlayerCode` and its DTO, then we should remove those files from the codebase.